src/test/java/pl/sda/library/service/AuthorServiceTest.java

```diff
@@ -83,6 +84,26 @@ public void should_return_changed_lastName_while_edit()  {
         assertEquals("DooEdit", afterEdit.getLastName());
     }
 
+    @Test
+    public void should_throw_AuthorDoesNotExistException_while_get_no_exist_author()  {
```

Comment:
should_throw_AuthorDoesNotExistException_while_get_no_exist_author -> should_throw_AuthorDoesNotExistException_when_author_does_not_exist

Reply:
aczkolwiek jesli zmienisz na zwracanie boolean to wtedy tez to inaczje bedzie wygladalo

natomiast w nazwach metod raczej uzywamy slowka when, a nie while

Reply:
ok